platforms/ios/src/TangramMap/TGHelpers.mm

```diff
@@ -49,8 +49,13 @@ + (NSError *)errorFromSceneError:(Tangram::SceneError)updateError
     NSString* value = [NSString stringWithUTF8String:updateError.update.value.c_str()];
     TGSceneUpdate* udpate = [[TGSceneUpdate alloc] initWithPath:path value:value];
```

Comment:
This is unused now, I think it would still be useful to somehow leave it in the `userInfo` error to know which update failed.